salt/_modules/cri.py

```diff
@@ -148,3 +148,22 @@ def wait_container(name, state, timeout=60, delay=5):
     else:
         log.error('Failed to find container "%s" in state "%s"', name, state)
         return False
+
+
+def component_is_running(name):
+    '''Return true the specified component is running.
+
+    .. note::
+
+       This uses the :command:`crictl` command, which should be configured
+       correctly on the system, e.g. in :file:`/etc/crictl.yaml`.
+    '''
+    log.info('Checking if compopent %s is running', name)
+    out = __salt__['cmd.run_all']('crictl pods -o json')
```

Comment:
wouldnt `crictl ps` be more exact as the pod may exist but it is in a crash loop

Reply:
Used the `crictl pods` but with extra params, as suggested by Nicolas.